diff --git a/mm/src/code/z_en_item00.c b/mm/src/code/z_en_item00.c
index 12233a36f..4c4051b99 100644
--- a/mm/src/code/z_en_item00.c
+++ b/mm/src/code/z_en_item00.c
@@ -919,7 +919,7 @@ s16 func_800A7650(s16 dropId) {
 
 Actor* Item_DropCollectible(PlayState* play, Vec3f* spawnPos, u32 params) {
     if (!(GameInteractor_Should(VB_ENEMY_DROP_COLLECTIBLE, true, *spawnPos))) {
-        return;
+        return (Actor*) NULL;;
     }
 
     s32 pad;
